client/components/Header/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { 
+	Route, Link 
```

Comment:
I'd keep this in one line for consistency. When our `import` block becomes huge, it's gonna be a problem navigating through it. Just stylistic, of course